node/src/wallets.rs

```diff
@@ -51,6 +50,12 @@ pub struct TxResponse {
     pub error: Option<BTreeMap<String, String>>,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct TxInfo {
+    pub txid: Txid,
```

Comment:
is the txid and confirmed enough information for the wallet? shouldn't it display at least sent or received amount in this tx?

Reply:
I've added some more data.